modules/iam-service-account/outputs.tf

```diff
@@ -50,3 +50,11 @@ output "service_account" {
   description = "Service account resource."
   value       = local.service_account
 }
+
+output "unique_id" {
+  description = "Fully qualified service account id."
+  value       = local.service_account.unique_id
+  depends_on = [
+    local.service_account
+  ]
```

Comment:
From my understanding, this should not be needed, because you are accessing `local.service_account.unique_id`, and this already creates implicit dependency to `local.service_account`

Reply:
yup, I just aligned to all other outputs

Reply:
removed